google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudConfigurable.java

```diff
@@ -92,12 +122,23 @@ public boolean canCheckConnection() {
     return false;
   }
 
-  @VisibleForTesting
-  CloudSdkPanel getCloudSdkPanel() {
-    return cloudSdkPanel;
-  }
-
-  private void createUIComponents() {
-    cloudSdkPanel = new CloudSdkPanel();
+  /**
+   * Checks for custom SDK validation errors and shows error label if they exist. To be removed once
+   * managed SDk feature is rolled out.
+   */
+  private void updateSdkValidationLabel() {
```

Comment:
is it possible to cover this in tests? like check for the visibility of the label depending on the validation status of the SDK.

Reply:
Sure, good idea!